fix: foreign key of the 1:1 and n:1 association is now using the defi… (#6197)
shopware/platform: changelog/_unreleased/2025-01-12-fix-foreign-key-definition.md

---
title: Fix foreign key definition
issue: 
author: Oliver Skroblin
author_email: oliver@goblin-coders.de
author_github: OliverSkroblin
---

# Core

> * Changed `AttributeEntityCompiler` to fix the foreign key definition when defining the fk column is defined.  

Reviewer: ```suggestion
* Changed `AttributeEntityCompiler` to fix the foreign key name, if a foreign key column is defined.
```

I had a few difficulties to understand what you want to described, hope I got it correct? :blush:

Author: FYI: externals can no approve suggestions 
<img width="369" alt="image" src="https://github.com/user-attachments/assets/99422c93-52ec-4c03-a0ea-0749553fe2da" />
